src/routes/components/dataTable/dataTable.tsx

```diff
@@ -97,7 +113,7 @@ class DataTable extends React.Component<DataTableProps, any> {
     });
   };
 
-  private handleOnSort = (event, index, direction) => {
+  private handleOnSort = (index, direction) => {
```

Comment:
looks like this is the cause of errors on sorting

```suggestion
  private handleOnSort = (event, index, direction) => {
```